pom.xml

```diff
@@ -13,7 +13,7 @@
   <!-- <artifactId>jenkinspider</artifactId> -->
   <artifactId>jenkinsci-appspider-plugin</artifactId>
   <!--<version>1.0</version>-->
-  <version>1.0.8-SNAPSHOT</version>
+  <version>1.0.9</version>
```

Comment:
Normally this is done using the maven release plugin. It'll do two things:
- Drop the `SNAPSHOT, commit, create GH release
- Increment version and add `-SNAPSHOT`, commit

It's worth noting that the 'official' source of this plugin (from jenkins GH org) https://github.com/jenkinsci/appspider-build-scanner-plugin is currently running 1.0.9-SNAPSHOT. 

This indicates that the Rapid7 code is behind. Normally with jenkins plugins they begin on some base repository, get forked to the jenkins org and follow up enhancements get made there. 

Unless I'm missing something, this change set should be targeted at the plugin hosted on jenkins GH org. Once merged there is a manual release process similar to what I mentioned about the maven release plugin above.